scripts/init-piwik.sh

```diff
@@ -56,6 +69,7 @@ if [ ! -z $DB_TABLES_PREFIX ]; then
   sed -i "/tables_prefix =/c\tables_prefix = \"$DB_TABLES_PREFIX\"" $CONFIG_FILE
 fi
 
+DB_CHARSET=utf8
```

Comment:
Did you mean to force the charset to utf8 here?  It was the default before, and could be overridden at runtime, but now its forced.  If you have a strong case for this, we can yank it from the readme and lose the if here.